src/frontend/src/components/CollapsibleInstructions/index.tsx

```diff
@@ -67,6 +79,25 @@ const CollapsibleInstructions = ({
               );
             })}
           </List>
+          {secondInstructionListTitle && (
+            <SecondInstructionsTitle
+              headerSize={headerSize}
+              marginBottom={InstructionsTitleMarginBottom}
+            >
+              {secondInstructionListTitle}
+            </SecondInstructionsTitle>
+          )}
+          {secondSetItems && (
+            <List ordered={ordered}>
+              {secondSetItems.map((item, index) => {
```

Comment:
This function can be pulled out into a variable for DRY

Reply:
done